RadarSDK/Radar.m

```diff
@@ -342,75 +350,67 @@ + (void)rejectEventId:(NSString *)eventId {
     [[RadarAPIClient sharedInstance] verifyEventId:eventId verification:RadarEventVerificationReject verifiedPlaceId:nil];
 }
 
-+ (void)sendEvent:(NSString *)customType
-     withMetadata:(NSDictionary *_Nullable)metadata
-completionHandler:(RadarSendEventCompletionHandler)completionHandler {
-    [self trackOnceWithCompletionHandler:^(RadarStatus status, CLLocation * _Nullable location, NSArray<RadarEvent *> * _Nullable events, RadarUser * _Nullable user) {
++ (void)sendLogConversionRequestWithType:(NSString * _Nonnull) type
+                                metadata:(NSDictionary * _Nullable) metadata
+                       completionHandler:(RadarLogConversionCompletionHandler) completionHandler {
+    [[RadarAPIClient sharedInstance] sendEvent:type withMetadata:metadata completionHandler:^(RadarStatus status, NSDictionary * _Nullable res, RadarEvent * _Nullable event) {
         if (status != RadarStatusSuccess) {
             if (completionHandler) {
                 [RadarUtils runOnMainThread:^{
-                    completionHandler(status, nil, nil, nil);
+                    completionHandler(status, nil);
                 }];
             }
 
             return;
         }
-
-        [[RadarAPIClient sharedInstance] sendEvent:customType withMetadata:metadata user:user trackEvents:events completionHandler:^(RadarStatus status, NSDictionary * _Nullable res, NSArray<RadarEvent *> * _Nullable events) {
-            if (status != RadarStatusSuccess) {
-                if (completionHandler) {
-                    [RadarUtils runOnMainThread:^{
-                        completionHandler(status, nil, nil, nil);
-                    }];
-                }
-
-                return;
-            }
-
-            if (completionHandler) {
-                [RadarUtils runOnMainThread:^{
-                    completionHandler(status, location, events, user);
-                }];
-            }
-        }];
+        
+        if (completionHandler) {
+            [RadarUtils runOnMainThread:^{
+                completionHandler(status, event);
+            }];
+        }
     }];
 }
 
-+ (void)sendEvent:(NSString *)customType
-     withLocation:(CLLocation *_Nullable)location
-         metadata:(NSDictionary *_Nullable)metadata
-completionHandler:(RadarSendEventCompletionHandler)completionHandler {
-    [self trackOnceWithLocation:location completionHandler:^(RadarStatus status, CLLocation * _Nullable location, NSArray<RadarEvent *> * _Nullable events, RadarUser * _Nullable user) {
++ (void)logConversionWithType:(NSString *)type
```

Comment:
Signatures look good - one small thing, we wanted to name the first param "name":

https://linear.app/radarlabs/issue/CORE-227#comment-1263621b

cc @david-goodfellow @jsani-radar